### ISSUE TITLE
toolbar-header is not draggable by default

### ISSUE THREAD

**Issue by parro-it** [connors/photon#22]
When using a frameless-window, only 
toolbar-footer is draggable, I think we have to add

```css
.toolbar-header {
   -webkit-app-region: drag;
}
```

By the way, photon is really awesome!

**Comment by connors** [comment]
I actually wanted to just add this the the base `.toolbar` class. The problem is that there are tons of conflicts with the default system behavior for dragging when you aren't using a frameless window. So fixing the frameless window breaks non-frameless windows.

Not sure how to fix it up so everything always plays nicely.

**Comment by developit** [comment]
@connors Just had a thought on this - what about tagging some element higher up the tree with a generic "make draggable things draggable" class?  Something like:

```html
<div class="allow-window-drag">
    <header class="toolbar toolbar-header">
        <h1 class="title">Draggable Bar</h1>
    </header>
</app>
```

**Comment by parro-it** [comment]
Or otherwise, just add a `draggable` class:

```html
    <header class="toolbar toolbar-header toolbar-draggable">
        <h1 class="title">Draggable Bar</h1>
    </header>
```

**Comment by connors** [comment]
@developit @parro-it I like that solution. I'll create a `.draggable` utility class so that you can add it as needed.

**Comment by developit** [comment]
Great, thanks, I'll add `<Toolbar draggable />` to [preact-photon](https://github.com/developit/preact-photon).

**Comment by connors** [comment]
Fixed in 3da1898b170cc28222fb0117cc31a8e39c9b8b3f
